src/django/api/models.py

```diff
@@ -422,6 +422,66 @@ class FacilityClaim(models.Model):
         choices=STATUS_CHOICES,
         default=PENDING,
         help_text='The current status of this facility claim')
+    approval_reason = models.TextField(
+        null=True,
+        blank=True,
+        help_text='The reason entered when approving this claim.')
+    approved_by = models.ForeignKey(
+        'User',
+        null=True,
+        on_delete=models.PROTECT,
+        help_text='The user who approved this facility claim',
+        related_name='approver_of_claim')
+    approval_date = models.DateTimeField(null=True)
+    denial_reason = models.TextField(
+        null=True,
+        blank=True,
+        help_text='The reason entered when denying this claim.')
+    denied_by = models.ForeignKey(
+        'User',
+        null=True,
+        on_delete=models.PROTECT,
+        help_text='The user who denied this facility claim.',
+        related_name='denier_of_claim')
+    denial_date = models.DateTimeField(null=True)
+    revocation_reason = models.TextField(
+        null=True,
+        blank=True,
+        help_text='The reason enetred when revoking this claim.')
+    revoked_by = models.ForeignKey(
+        'User',
+        null=True,
+        on_delete=models.PROTECT,
+        help_text='The user who revoked this facility claim',
+        related_name='revoker_of_claim')
+    revocation_date = models.DateTimeField(null=True)
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    history = HistoricalRecords()
+
+
+class FacilityClaimReviewNote(models.Model):
+    """
+    A note entered by an administrator when reviewing a FacilityClaim.
+    """
+
+    claim = models.ForeignKey(
+        'FacilityClaim',
+        null=False,
+        on_delete=models.PROTECT,
+        help_text='The facility claim for this note'
+    )
+    author = models.ForeignKey(
+        'User',
+        null=False,
+        on_delete=models.PROTECT,
+        help_text='The author of the facility claim review note')
+    note = models.TextField(
+        null=False,
+        blank=False,
+        help_text='The review note')
```

Comment:
This needs a `history = HistoricalRecords()` setting, too.

Reply:
I checked and it's there -- https://github.com/open-apparel-registry/open-apparel-registry/blob/1116b620d3870d8b92d6b4da81588bf91dc85705/src/django/api/models.py#L489 -- but just not showing up in the diff.